doc/users/pyplot_tutorial.rst

```diff
@@ -7,10 +7,11 @@ Pyplot tutorial
 :mod:`matplotlib.pyplot` is a collection of command style functions
 that make matplotlib  work like MATLAB.
 Each ``pyplot`` function makes
-some change to a figure: e.g., create a figure, create a plotting area
-in a figure, plot some lines in a plotting area, decorate the plot
-with labels, etc....  :mod:`matplotlib.pyplot` is stateful, in that it
-keeps track of the current figure and plotting area, and the plotting
+some change to a figure: e.g., creates a figure, creates a plotting area
+in a figure, plots some lines in a plotting area, decorates the plot
+with labels, etc.  In :mod:`matplotlib.pyplot` various states are preserved
```

Comment:
I would be a bit more explicit about what state is tracked which is only the current figure and axes (there is also a `gci`, but it should not be used).